cloudprem/modules/webhooks/main.tf

```diff
@@ -0,0 +1,333 @@
+
+  namespace = "default"
+
+  timeout = 600
+  reuse_values = true
+//  force_update = true
+//  cleanup_on_fail = true
+
+  set_sensitive {
+    name = "event-service.frontegg.clientId"
+    value = "dummyid"
+  }
+  set_sensitive {
+    name = "event-service.frontegg.apiKey"
+    value = "dummyid"
+  }
+
+  set {
+    name  = "webhook-service.messageBroker.brokerList"
+    value = replace(aws_msk_cluster.this.zookeeper_connect_string,",","\\,")
+  }
+  set {
+    name  = "event-service.messageBroker.brokerList"
+    value = replace(aws_msk_cluster.this.zookeeper_connect_string,",","\\,")
+  }
+  set {
+    name  = "integrations-service.messageBroker.brokerList"
+    value = replace(aws_msk_cluster.this.zookeeper_connect_string,",","\\,")
+  }
+  set {
+    name  = "connectors-worker.messageBroker.brokerList"
+    value = replace(aws_msk_cluster.this.zookeeper_connect_string,",","\\,")
+  }
+
+  set {
+    name = "webhook-service.mongo.connectionString"
+    value = "mongodb://foo:barbut8chars@${aws_docdb_cluster.this.endpoint}:${aws_docdb_cluster_instance.cluster_instances.0.port}/?retryWrites=false"
+  }
+  set {
+    name = "integrations-service.mongo.connectionString"
+    value = "mongodb://foo:barbut8chars@${aws_docdb_cluster.this.endpoint}:${aws_docdb_cluster_instance.cluster_instances.0.port}/?retryWrites=false"
+  }
+
+  set {
+    name = "webhook-service.redis.host"
+    value = aws_elasticache_cluster.redis.cache_nodes.0.address
+  }
+  set {
+    name = "event-service.redis.host"
+    value = aws_elasticache_cluster.redis.cache_nodes.0.address
+  }
+  set {
+    name = "connectors-worker.redis.host"
+    value = aws_elasticache_cluster.redis.cache_nodes.0.address
+  }
+
+  set {
+    name = "webhook-service.mysql.host"
+    value = var.rds_address
+  }
+  set {
+    name = "webhook-service.mysql.username"
+    value = var.rds_user
+  }
+
+  set_sensitive {
+    name = "webhook-service.mysql.password"
+    value = var.rds_pass
+  }
+  set {
+    name = "event-service.database.host"
+    value = var.rds_address
+  }
+  set {
+    name = "event-service.database.username"
+    value = var.rds_user
+  }
+
+  set_sensitive {
+    name = "event-service.database.password"
+    value = var.rds_pass
+  }
+
+  set {
+    name = "api-gateway.frontegg.authenticationPublicKey"
+    value = data.kubernetes_secret.frontegg.data.pubkey
+  }
+
+  set {
+    name = "frontegg.images.username"
+    value = data.kubernetes_secret.frontegg.data.username
+  }
+  set_sensitive {
+    name = "frontegg.images.password"
+    value = data.kubernetes_secret.frontegg.data.password
+  }
+
+  set {
+    name = "integrations-service.frontegg.slack.encryptionKey"
+    value = "dummy"
+  }
+  set {
+    name = "connectors-worker.frontegg.channels"
+    value = "slack"
+  }
+  set {
+    name = "connectors-worker.frontegg.emails.provider"
+    value = "sendgrid"
+  }
+  set {
+    name = "connectors-worker.frontegg.emails.sendgrid.apiKey"
+    value = "dummyval"
+  }
```

Comment:
Just to ask, this looks really boilerplatey. Any way we can use a `value.yml` with the helm provisioner? Just to help clean up all these individual `set` blocks? Or does the `set_sensitive` have to happen to guard secrets? Maybe we could come up a hybrid approach, of the "plaintext" values in their own file, but the sensitive values blocked out here?

Not a blocker, just asking...

Reply:
Yup, it's now in a values.yaml :)

Reply:
Nice! :+1: